lemur/static/app/angular/app.js

```diff
@@ -27,17 +27,23 @@ var lemur = angular
         templateUrl: 'angular/welcome/welcome.html'
       });
 
-    $authProvider.oauth2({
-      name: 'example',
-      url: 'http://localhost:8000/api/1/auth/ping',
-      redirectUri: 'http://localhost:3000/',
-      clientId: 'client-id',
-      responseType: 'code',
-      scope: ['openid', 'email', 'profile', 'address'],
-      scopeDelimiter: ' ',
-      authorizationEndpoint: 'https://example.com/as/authorization.oauth2',
-      requiredUrlParams: ['scope']
-    });
+    active_providers = AuthenticationService.get_providers();
```

Comment:
Because get_providers returns a 'promise' or async result you probably want to wrap your logic after the providers have returned: 

```
AuthenticationService.get_providers().then(function (providers) {
    // do $authProvider manipulation
});
```
Otherwise when you call ```active_providers = AuthenticationService.get_providers();``` ```active_providers``` will be undefined until some later point